be/src/exprs/bitmap_functions.cpp

```diff
@@ -675,4 +675,60 @@ StatusOr<ColumnPtr> BitmapFunctions::bitmap_subset_in_range(FunctionContext* con
     return builder.build(ColumnHelper::is_all_const(columns));
 }
 
+StatusOr<ColumnPtr> BitmapFunctions::bitmap_to_binary(FunctionContext* context, const starrocks::Columns& columns) {
+    ColumnViewer<TYPE_OBJECT> viewer(columns[0]);
+
+    size_t size = columns[0]->size();
+    ColumnBuilder<TYPE_VARCHAR> builder(size);
+
+    std::unique_ptr<char[]> buf;
+    size_t cur_buf_size = 0;
+
+    for (int row = 0; row < size; ++row) {
+        BitmapValue* bitmap = viewer.value(row);
+        int serialize_size = bitmap->getSizeInBytes();
+        if (serialize_size > cur_buf_size) {
+            buf.reset(new char[serialize_size]);
+            cur_buf_size = serialize_size;
+        }
+
+        bitmap->write((char*)buf.get());
+        builder.append(Slice(buf.get(), serialize_size));
+    }
+    ColumnPtr col = builder.build(ColumnHelper::is_all_const(columns));
+    if (col->capacity_limit_reached()) {
+        return Status::InternalError("Size of binary column generated by bitmap_to_binary reaches limit (4G)");
+    } else {
+        return col;
+    }
+}
+
+StatusOr<ColumnPtr> BitmapFunctions::bitmap_from_binary(FunctionContext* context, const Columns& columns) {
+    ColumnViewer<TYPE_VARCHAR> viewer(columns[0]);
+    size_t size = columns[0]->size();
+    ColumnBuilder<TYPE_OBJECT> builder(size);
+
+    for (int row = 0; row < size; ++row) {
+        if (viewer.is_null(row)) {
+            builder.append_null();
+            continue;
+        }
+
+        auto src_value = viewer.value(row);
+        if (src_value.size == 0) {
+            builder.append_null();
+            continue;
+        }
+
+        BitmapValue bitmap;
+        bool res = bitmap.valid_and_deserialize(src_value.data, src_value.size);
+        if (!res) {
+            return Status::InternalError("Invalid binary format, unable to deserialize the binary to bitmap");
```

Comment:
When the input is empty, the returned value is null. But the empty is one kind of invalid input to me.
But for other invalid inputs, error messages returned.
My question is that this behavior does not consist of different invalid inputs.
My suggestion is that
1. return an error message when the input is empty neither;
or 2. return null for all invalid inputs.

Reply:
The purpose of writing this was to have the same behavior as bitmap_from_base64. Returning an error is more reasonable

Reply:
In order to be consistent with Base64 processing, null is returned uniformly.